### PR TITLE
Small touches towards a homogeneous CLI

### PR DESCRIPTION
**Pull request by ktsakalozos**
Changes:
- remove global flags -d and -v
- flag descriptions should start with lowercase and should not end with a '.'
- command descriptions should start with a capital letter and should not end with a '.'
- error messages that suggest you to run a command should prefix the command with an empty line and double space and suffix the command with an empty line
- minor tweaks in the command descriptions
- few spelling errors

### CODE REVIEW

File: src/k8s/cmd/k8s/k8s_disable.go [hunk 1]
```diff
@@ -14,8 +14,8 @@ import (
 func newDisableCmd() *cobra.Command {
 	disableCmd := &cobra.Command{
 		Use:               "disable <functionality>",
-		Short:             "Disable a specific functionality in the cluster",
-		Long:              fmt.Sprintf("Disable one of the specific functionalities: %s.", strings.Join(componentList, ",")),
+		Short:             "Disable cluster core functionalities",
```
Comment (eaudetcobello) [inline src/k8s/cmd/k8s/k8s_disable.go:17]:
```suggestion
		Short:             "Disable core cluster functionalities",
```

File: src/k8s/cmd/k8s/k8s_enable.go [hunk 2]
```diff
@@ -18,8 +18,8 @@ var (
 func newEnableCmd() *cobra.Command {
 	enableCmd := &cobra.Command{
 		Use:               "enable <functionality>",
-		Short:             "Enable a specific functionality in the cluster",
-		Long:              fmt.Sprintf("Enable one of the specific functionalities: %s.", strings.Join(componentList, ", ")),
+		Short:             "Enable cluster core functionalities",
```
Comment (eaudetcobello) [inline src/k8s/cmd/k8s/k8s_enable.go:21]:
```suggestion
		Short:             "Enable core cluster functionalities",
```

File: src/k8s/cmd/k8s/k8s_status.go [hunk 3]
```diff
@@ -59,8 +59,8 @@ func newStatusCmd() *cobra.Command {
 			return f.Print(clusterStatus)
 		},
 	}
-	statusCmd.PersistentFlags().StringVar(&statusCmdOpts.outputFormat, "format", "plain", "Specify in which format the output should be printed. One of plain, json or yaml")
-	statusCmd.PersistentFlags().DurationVar(&statusCmdOpts.timeout, "timeout", 90*time.Second, "The max time to wait for the K8s API server to be ready.")
-	statusCmd.PersistentFlags().BoolVar(&statusCmdOpts.waitReady, "wait-ready", false, "If set, the command will block until at least one cluster node is ready.")
+	statusCmd.PersistentFlags().StringVar(&statusCmdOpts.outputFormat, "format", "plain", "specify in which format the output should be printed. One of plain, json or yaml")
```
Comment (eaudetcobello) [inline src/k8s/cmd/k8s/k8s_status.go:62]: Is the default also plain here ?

```suggestion
	statusCmd.PersistentFlags().StringVar(&statusCmdOpts.outputFormat, "format", "plain", "specify in which format the output should be printed. One of plain (default), json or yaml")
```

**Review by eaudetcobello — COMMENTED** [review]
Nice!

File: src/k8s/cmd/k8s/k8s_status.go (hunk 3 above)
Comment (ktsakalozos) [reply src/k8s/cmd/k8s/k8s_status.go:62]: We have a PR from @bschimke95 with work on the status command.

**Review by bschimke95 — APPROVED** [review]
LGTM

File: src/k8s/cmd/k8s/k8s_get_join_token.go [hunk 4]
```diff
@@ -60,7 +60,7 @@ func newGetJoinTokenCmd() *cobra.Command {
 		},
 	}
 
-	getJoinTokenCmd.PersistentFlags().StringVarP(&getJoinTokenCmdOpts.outputFormat, "output-format", "o", "plain", "Specify in which format the output should be printed. One of plain, json or yaml")
+	getJoinTokenCmd.PersistentFlags().StringVarP(&getJoinTokenCmdOpts.outputFormat, "output-format", "o", "plain", "set the output format to one of plain (default), json or yaml")
```
Comment (neoaggelos) [inline src/k8s/cmd/k8s/k8s_get_join_token.go:63]:
```suggestion
	getJoinTokenCmd.PersistentFlags().StringVarP(&getJoinTokenCmdOpts.outputFormat, "output-format", "o", "plain", "set the output format to one of plain, json or yaml")
```

Cobra will note the default value already:
```
Create a join token for a node to join the cluster

Usage:
  k8s get-join-token <node-name> [flags]

Flags:
  -h, --help                   help for get-join-token
  -o, --output-format string   Specify in which format the output should be printed. One of plain, json or yaml (default "plain")
      --worker                 generate a join token for a worker node
```

**Review by neoaggelos — APPROVED** [review]
Just one comment

File: src/k8s/cmd/k8s/k8s_get_join_token.go (hunk 4 above)
Comment (louiseschmidtgen) [reply src/k8s/cmd/k8s/k8s_get_join_token.go]: Can we check that the flags are in the correct order for all commands? Can we make sure the `--help` flag is always the first. (I believe there might be some inconsistencies across commands)

File: src/k8s/cmd/k8s/k8s_bootstrap.go [hunk 5]
```diff
@@ -65,8 +65,8 @@ func newBootstrapCmd() *cobra.Command {
 		},
 	}
 
-	bootstrapCmd.PersistentFlags().BoolVar(&bootstrapCmdOpts.interactive, "interactive", false, "Interactively configure the most important cluster options.")
-	bootstrapCmd.PersistentFlags().DurationVar(&bootstrapCmdOpts.timeout, "timeout", 90*time.Second, "The max time to wait for k8s to bootstrap.")
+	bootstrapCmd.PersistentFlags().BoolVar(&bootstrapCmdOpts.interactive, "interactive", false, "interactively configure the most important cluster options")
+	bootstrapCmd.PersistentFlags().DurationVar(&bootstrapCmdOpts.timeout, "timeout", 90*time.Second, "the max time to wait for k8s to bootstrap")
```
Comment (louiseschmidtgen) [inline src/k8s/cmd/k8s/k8s_bootstrap.go:69]:
```suggestion
	bootstrapCmd.PersistentFlags().DurationVar(&bootstrapCmdOpts.timeout, "timeout", 90*time.Second, "the maximum time to wait for k8s to bootstrap in seconds")
```

Comment (louiseschmidtgen) [reply src/k8s/cmd/k8s/k8s_bootstrap.go:69]: Should we add a time expectation here? Is the default time shown to the user?

Comment (louiseschmidtgen) [reply src/k8s/cmd/k8s/k8s_bootstrap.go:69]: Is it normal to use abbreviation, such as max?

**Review by louiseschmidtgen — APPROVED** [review]
LGTM

File: src/k8s/cmd/k8s/k8s_bootstrap.go (hunk 5 above)
Comment (bschimke95) [reply src/k8s/cmd/k8s/k8s_bootstrap.go]: > Should we add a time expectation here? Is the default time shown to the user?

This greatly depends on your machine and the current workloads.
e.g. it takes ~5s for on a idle machine but ~30s? when running something heavy in the background.

Comment (louiseschmidtgen) [reply src/k8s/cmd/k8s/k8s_bootstrap.go]: That is true.

**Pull request merged**